routes/webhook/github.rb

```diff
@@ -40,9 +40,8 @@ def handle_installation(data)
     installation = GithubInstallation[installation_id: data["installation"]["id"]]
     case data["action"]
     when "deleted"
-      unless installation
-        return error("Unregistered installation")
-      end
+      return error("Unregistered installation") unless installation
+      return error("Inactive project") unless installation.project.active?
```

Comment:
This will return an error both for non-`visible` projects as well as projects with suspended accounts. If we want to do this only for projects with suspended accounts, I would add `Project#suspended?` and call that instead of `active?`. However, if you are OK doing it for both, this seems fine.